wallet/wallet.go

```diff
@@ -657,10 +665,13 @@ func (wt *Wallet) SyncTx(order int64) (clijson.BlockHttpResult, error) {
 			return block, err
 		}else{
 			if isBlue != "1"{
```

Comment:
isBule should be `boolean`, refactoring the rpc calling method instead of using "1" at this level.